_component_design/buttons.md

```diff
@@ -14,203 +14,193 @@ last-modified: 2017-08-17
-{% figure [caption:"Regular button redlines"] %}
-![]({{site.cdn_url}}/img/components/Buttons/buttons-specs-regular.svg)
+{% column right:"hxCol-8 hxCol-xs-12 hxCol-sm-12 hxCol-md-8 hxCol-lg-8" %}
+{% figure [caption:"Button spacing specs"] [class:"image bg-light border"] %}
+![]({{site.url}}/assets/images/components/inputs-and-controls/buttons/buttons-tertiary.svg){:width="100%"}
 {% endfigure %}
 {% endcolumn %}
 </div>
 
-## Large Button Specs
-
+### Loading states
+{: .hxSubSectionTitle}
 <div class="hxRow">
-{% column left:"hxCol" %}
-- `font-family:roboto-medium`
-- `font-size:16px`
-- `text-transform:uppercase`
-- `border-radius:2px`
-{% endcolumn %}
-{% column right:"hxCol" %}
-{% figure [caption:"Large button redlines"] %}
-![]({{site.cdn_url}}/img/components/Buttons/buttons-specs-large.svg)
-{% endfigure %}
-{% endcolumn %}
-</div>
+{% column left:"hxCol-4 hxCol-xs-12 hxCol-sm-12 hxCol-md-4 hxCol-lg-4" %}
 
-## Compact Button Specs
+- Loading icon animation should rotate time: 0.8 sec/revolution.
+- For further information regarding the loading pattern, see the [loading pattern]({{site.baseurl}}/components/loading.html).
 
-<div class="hxRow">
-{% column left:"hxCol" %}
-- `font-family:roboto-medium`
-- `font-size:10px`
-- `text-transform:uppercase`
-- `border-radius:2px`
-{% endcolumn %}
-{% column right:"hxCol" %}
-{% figure [caption:"Compact button redlines"] %}
-![]({{site.cdn_url}}/img/components/Buttons/buttons-specs-compact.svg)
-{% endfigure %}
-{% endcolumn %}
-</div>
 
-# Guidance
+{% endcolumn %}
 
--   Generally, use primary buttons for actions that go to the next step and use
-    secondary buttons for actions that happen on the current page.
+{% column right:"hxCol-8 hxCol-xs-12 hxCol-sm-12 hxCol-md-8 hxCol-lg-8" %}  
+{% figure [caption:"Button (loading state) spacing"] [class:"image bg-light border"] %}
+![]({{site.url}}/assets/images/components/inputs-and-controls/buttons/buttons-loading-1.svg){:width="100%"}
+{% endfigure %}
+{% figure [caption:"Button (loading state) specifications for primary, secondary, and tertiary weights."] [class:"image bg-light border"] %}
+![]({{site.url}}/assets/images/components/inputs-and-controls/buttons/buttons-loading-2.svg){:width="100%"}
+{% endfigure %}
+{% endcolumn %}
 
--   Style the button most users should click in a way that distinguishes from
-    other buttons on the page. Try using the "large button" or the most
-    visually distinct fill color.
+</div>
 
--   Make sure buttons should look clickable—use color variations to distinguish
-    static, hover and active states.
+### Button sets
+{: .hxSubSectionTitle}
+<div class="hxRow">
+{% column left:"hxCol-4 hxCol-xs-12 hxCol-sm-12 hxCol-md-4 hxCol-lg-4" %}
 
--   Avoid using too many buttons on a page.
+- Buttons should be aligned in order of Primary, Secondary, Tertiary (cancel).
+- Button sets are most often used in Popovers, Modals or Create/Edit Forms to group the submit, back/next, and cancel buttons.
```

Comment:
lowercase for popovers...

Uppercase and bold for Submit, Back/Next, and Cancel

Reply:
Updated to `the submit, back or next, and cancel buttons.`